src/components/Organizations/AddUserToOrganization/index.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import { Mutation } from 'react-apollo';
+import ReactSelect from 'react-select';
+
+import Button from 'components/Button';
+// @TODO: add this once the logic exists
+import withLogic from 'components/Organizations/AddUserToOrganization/logic';
+import gql from 'graphql-tag';
+
+import { Footer } from '../SharedStyles';
+import { NewUser, OrganizationSelect } from './Styles';
+
+const ADD_USER_MUTATION = gql`
+  mutation AddUserToOrganization($email: String!, $organization: Int!, $owner: Boolean) {
+    addUserToOrganization(input: { user: { email: $email }, organization: $organization, owner: $owner }) {
+      id
+    }
+  }
+`;
+
+/**
+ * Confirms the deletion of the specified name and type.
+ */
+export const AddUserToOrganization = ({
+  organization,
+  close,
+  inputValueEmail,
+  setInputValue,
+  checkboxValueOwner,
+  setCheckboxValueOwner,
+  onAddUser,
+}) => {
+  const [selectedOrganization, setSelectedOrganization] = useState(organization);
+
+  let options = [
+    {
+      label: organization && organization.name,
+      value: organization && organization.id,
+    },
+  ];
+
+  return (
+    <Mutation mutation={ADD_USER_MUTATION} onError={err => console.error(err)}>
+      {(addUser, { error, data }) => {
+        if (error) {
+          return <div>{error.message}</div>;
+        }
+        if (data) {
+          onAddUser().then(() => {
+            setInputValue({ target: { value: '' } });
+            close();
+          });
+        }
+        return (
+          <NewUser>
+            <h4>Add User</h4>
+            <div className="form-box">
+              <label>
+                User name: <span style={{ color: '#E30000' }}>*</span>
+                <input
+                  className="inputEmail"
+                  type="text"
+                  placeholder="Enter email"
+                  value={inputValueEmail}
+                  onChange={setInputValue}
+                />
+              </label>
+            </div>
+            <label>
```

Comment:
I don't think this selection is required because it will only ever be the one organization you can add the user to within the context of the organization

Reply:
ah, makes sense